jaeger-core/src/main/java/com/uber/jaeger/Span.java

```diff
@@ -117,8 +118,18 @@ public String getOperationName() {
   @Override
   public Span setBaggageItem(String key, String value) {
```

Comment:
`setBaggageItem` should check that `key` is not null or empty and `value` isn't null.